R/tax_unique.R

```diff
@@ -29,11 +29,16 @@
 #' occurrence dataframe is of this type.
 #' @param resolution \code{character}. The taxonomic resolution at which to
 #' identify unique occurrences, either "species" (the default) or "genus".
+#' @param orig \code{logical}. Should the original dataframe be returned with
```

Comment:
Probably doesn't matter too much, but maybe we should change the name of this argument? Maybe something like bind? I only say this because we have a similar argument name for origin in another function.